docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.7"
+services:
+  nginx:
+    build:
+      context: infra/nginx
+      dockerfile: Dockerfile
```

Comment:
And also Minio below:

Is there a reason why we can't/shouldn't use the stock images? I usually bind mount + use env vars for these, but that's with other applications, and not Nginx/Minio in particular.

Reply:
This way we can package them with all the files that they need to run on their own. No need to bind mount when it already has the files in the image.

Reply:
This also means that we don't have to deploy the file changes to whatever is serving the bind mount. The config+deploys happens in a full package.